go.mod

```diff
@@ -1,44 +1,68 @@
 module github.com/moby/vpnkit
 
-go 1.16
+go 1.17
 
 require (
 	github.com/Microsoft/go-winio v0.4.11
+	github.com/google/uuid v0.0.0-20170814143639-7e072fc3a7be
```

Comment:
this one has tagged releases, and (I may be mistaken ) but there may be security issues with older version